infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -69,6 +70,21 @@
 
     private Orchestrator orchestrator;
 
+    /**
+     * System down handler to break out of live-locks if the runtime cannot reach the cluster for a
+     * certain amount of time.
+     */
+    private final Runnable runtimeSystemDownHandler = () -> {
+        log.warn("ManagementServer: Runtime stalled. Invoking systemDownHandler.");
```

Comment:
Would it be possible (and beneficial) to have more descriptive log message about trying to break out of live-lock rather than pointing to the mechanics of how we are doing it?

Also we are only logging and throwing an exception in the scope this runnable however the log implies of a handler being invoked which might be confusing.

Reply:
I'll add a descriptive log.